x/staging/posts/types/msgs.go

```diff
@@ -51,24 +49,23 @@ func (msg MsgCreatePost) ValidateBasic() error {
 	}
 
 	if len(strings.TrimSpace(msg.Message)) == 0 && len(msg.Attachments) == 0 && msg.Poll == nil {
-		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest,
-			"post message, attachments or poll are required and cannot be all blank or empty")
+		return ErrInvalidEmptyFields
 	}
 
 	if !subspacestypes.IsValidSubspace(msg.Subspace) {
-		return sdkerrors.Wrap(ErrInvalidSubspace, "post subspace must be a valid sha-256 hash")
+		return subspacestypes.ErrInvalidSubspaceID
```

Comment:
It's always better to wrap the wrong value
```suggestion
		return sdkerrors.Wrap(subspacestypes.ErrInvalidSubspaceID, msg.Subspace)
```